maps/RandomZLevels/wildwest_props.dm

```diff
@@ -0,0 +1,324 @@
+//Turf
+/turf/unsimulated/wall/meat
+	name = "?"
+	desc = null
+	icon = 'icons/turf/meat.dmi'
+	icon_state = "meat255"
+
+/turf/unsimulated/wall/meat/canSmoothWith()
+	return null
+
+/turf/unsimulated/wall/guts
+	name = "guts"
+	desc = "Some kind of twisting intestinal layers."
+	icon = 'icons/turf/meat.dmi'
+	icon_state = "guts0"
+	walltype = "guts"
+
+/turf/unsimulated/wall/guts/canSmoothWith()
+	var/static/list/smoothables = list(/turf/unsimulated/wall/guts)
+	return smoothables
+
+/turf/simulated/floor/plating/flesh
+	name = "?"
+	desc = null
+	icon = 'icons/turf/meat.dmi'
+	icon_state = "flesh"
+
+/turf/simulated/floor/plating/flesh/New()
+	..()
+	var/image/img = image('icons/turf/rock_overlay.dmi', "flesh_overlay",layer = SIDE_LAYER)
+	img.pixel_x = -4*PIXEL_MULTIPLIER
+	img.pixel_y = -4*PIXEL_MULTIPLIER
+	img.plane = BELOW_TURF_PLANE
+	overlays += img
+
+//Objects
+/obj/item/voucher/free_item/scrip
+	name = "scrip"
+	desc = "Redeem at a Deepvein Trust vendor."
+	freebies = list()
+	vend_amount = 1
+	single_items = 1
+	shred_on_use = 1
+
+/obj/item/voucher/free_item/scrip/liberator
+	name = "liberator scrip"
+	freebies = list(/obj/item/weapon/gun/energy/laser/liberator)
+
+/obj/item/voucher/free_item/scrip/drill
+	name = "drill scrip"
+	freebies = list(/obj/item/weapon/pickaxe/drill)
+
+/obj/item/voucher/free_item/scrip/lazarus
+	name = "lazarus scrip"
+	freebies = list(/obj/item/weapon/lazarus_injector)
+
+/obj/item/voucher/free_item/scrip/rifle
+	name = "rifle scrip"
+	freebies = list(/obj/item/weapon/gun/projectile/hecate/hunting)
+
+/obj/item/voucher/free_item/scrip/sausage
+	name = "sausage scrip"
+	freebies = list(/obj/item/weapon/reagent_containers/food/snacks/sausage)
+
+/obj/item/voucher/free_item/scrip/threefiftyseven
+	name = ".357 scrip"
+	freebies = list(/obj/item/ammo_storage/box/a357)
+
+/obj/machinery/vending/deepvein
+	name = "\improper Deepvein Trust Company Store"
+	desc = "Use your 'wages' here!"
+	product_slogans = list(
+		"Please have your scrip ready for vending."
+	)
+	product_ads = list(
+		"Time is money, so get back to digging!"
+	)
+	vend_reply = "What a glorious time to mine!"
+	icon_state = "mining"
+	vouched = list(
+		/obj/item/weapon/pickaxe/drill = 10,
+		/obj/item/weapon/lazarus_injector = 10,
+		/obj/item/weapon/gun/energy/laser/liberator = 10,
+		/obj/item/weapon/gun/projectile/hecate/hunting = 10,
+		/obj/item/weapon/reagent_containers/food/snacks/sausage = 10,
+		/obj/item/ammo_storage/box/a357 = 20
+		)
```

Comment:
All the other closing parentheses are unindented after the lists. Yes, this is very much a nitpick.